contrib/systemd/containerized-installs/openvswitch.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
```

Comment:
Actually, we're writing to /etc/systemd/system which is higher weighting than /usr/lib/systemd/system where rpms would be installed. Can we just rely on that? If not should I also prefix the node and master services as they could install both the rpms and containers.

We should totally just ship containers via rpms so we can conflict at that level. :laughing: